behave/features/cla_fala/fala_end_to_end_tests.feature

```diff
@@ -6,15 +7,25 @@ Feature: FALA end to end tests
   Background: Homepage page
     Given I am on the Find a legal aid adviser homepage
 
-  @fala-search-location @a11y-check
-  Scenario Outline: Search for legal advisers via postcode and city
+  @fala-search-postcode @a11y-check
+  Scenario Outline: Search for legal advisers via postcode
     Given I provide the "<location>" details
     When I select the "search" button on the FALA homepage
     Then I am taken to the page corresponding to "<location>" result
     Examples:
       | location |
       | SW1H 9AJ |
+
+
+  @fala-search-city @a11y-check
+  Scenario Outline: Search for legal advisers via invalid locations returns error
```

Comment:
Incorrect postcode is also being tested, so just mention it in the Scenario Outline here.